src/terra-dev-site/about/WhatIsTerra.a.about.mdx

```diff
@@ -18,3 +19,16 @@ Check out the [Getting Started Guide](/guides/terra-ui/installing-components) to
 <MedicineIcon />
 </div>
 </div>
+
+## Contributing
+
+<div className="tui-illustration-grid">
+<div className="tui-illustration-grid-col">
+
+Our [contribution guide](/about/terra-ui/contributing/contribution-guidelines) explains the project's contribution process and standards.
+
+</div>
+<div className="tui-illustration-grid-col tui-illustration-grid-col-illustration">
```

Comment:
Something is happening with the css (haven't investigated further) but the new image and column is not responsively resizing the same at the top one/existing SVG illustration is, to keep it inline and fully visible (notice reviewing on mobile):
![Screen Shot 2022-03-04 at 12 23 34 PM](https://user-images.githubusercontent.com/1355485/156820632-f092a9d3-b727-44bb-9d0f-bdcae56e8928.png)